### PR TITLE
JS context menu polish

### PR DESCRIPTION
**Pull request by saulmc**
- fixes horizontal positioning
- centers reactions display
- increases menu width
- normalizes spacing

### CODE REVIEW

File: components/Chat/Message/MessageReactionsList.tsx [hunk 1]
```diff
@@ -210,14 +212,20 @@ const MessageReactionsListInner: FC<MessageReactionsListProps> = ({
...
       {list.length !== 0 ? (
-        <FlatList
-          contentContainerStyle={styles.reactionsContainer}
-          data={list}
-          horizontal
-          renderItem={renderItem}
-          keyExtractor={keyExtractor}
-          showsHorizontalScrollIndicator={false}
-        />
+        <Portal>
```
Comment (nmalzieu) [inline components/Chat/Message/MessageReactionsList.tsx:215]: I don't understand the use of Portal here. Isn't Portal used to render somewhere else but it needs a PortalHost?

**Review by nmalzieu — CHANGES_REQUESTED** [review]

File: components/Chat/Message/MessageReactionsList.tsx (hunk 1 above)
Comment (nmalzieu) [reply components/Chat/Message/MessageReactionsList.tsx:215]: @saulmc 
Allright not sure I know how this works
Documentation at https://callstack.github.io/react-native-paper/docs/components/Portal/ says
> It requires a [Portal.Host](https://callstack.github.io/react-native-paper/docs/components/Portal/PortalHost) component to be rendered somewhere in the parent tree
and by checking out your branch I don't see PortalHost anywhere
Am I misunderstanding the doc?

Comment (saulmc) [reply components/Chat/Message/MessageReactionsList.tsx:215]: I think it is included in the PortalProvider in App.tsx. From the docs:

> Note that if you're using the Provider component, this already includes a Portal.Host.

Comment (saulmc) [reply components/Chat/Message/MessageReactionsList.tsx:215]: Also, I don't really know how this works either -- just copying a pattern that Alex had used for the message bubble.

Comment (nmalzieu) [reply components/Chat/Message/MessageReactionsList.tsx:215]: got it. Let's go then!

**Review by nmalzieu — APPROVED** [review]

**Pull request merged**